modules/contact/src/MechanicalContactConstraint.C

```diff
@@ -368,7 +591,7 @@ MechanicalContactConstraint::computeContactForce(PenetrationInfo * pinfo)
               else // treat as frictionless
                 pinfo->_contact_force = contact_force_normal;
             }
-            if (capacity == 0)
+            if (MooseUtils::absoluteFuzzyEqual(capacity, 0))
```

Comment:
I'm not sure that making these fuzzy here is the right thing to do.  `capacity` is set to exactly zero if it should be zero.  Was this important for getting a model to work correctly?